### ISSUE TITLE
Consider converting servlets into Spring MVC Controllers

### ISSUE THREAD

**Issue by jzheaux** [terracotta-bank/terracotta-bank#9]
Controllers are a bit more familiar these days, and it may be helpful to favor them in the app since we are already using Spring Boot.

By converting over to controllers it will also be simpler to use tools like MockMvc to simplify some of the unit tests Terracotta uses to demonstrate vulnerabilities.

**Comment by tummala** [comment]
Yes, using Spring controllers are configured via IOC like any other objects. This makes them easy to test and integrate with other objects.
